app/src/components/dialog/DialogBase.js

```diff
@@ -41,8 +51,8 @@ class DialogBase extends Component {
             title={title}
             actions={actions}
             modal={false}
-            contentStyle={contentStyle || {}}
-            bodyStyle={bodyStyle || {}}
+            contentStyle={{...styles.contentStyle, ...contentStyle} || styles.contentStyle}
```

Comment:
Spreading the styles.contentStyle has the same effect as the || when contentStyle is not defined. The `|| styles.contentStyle` will never be executed. Currently this works because babel translates it to `Object.extend` which is why it will work when contentStyle is undefined. (If/when it where to use a Iterator, like the array one does, you'll need to set a defaultProp to not get an error. 

Same for the next line.